### ISSUE TITLE
can_buy does not work for me?

### ISSUE THREAD

**Issue by ThatGuyNathan54** [goblivend/the-farmer-was-replaced#1]
![image_2024-05-19_043636221](https://github.com/goblivend/the-farmer-was-replaced/assets/53066406/d2c3ab09-c931-4120-8f93-b2eaf5e7f430)

**Comment by goblivend** [comment]
yep this was for an old version of the game that didn't have dictionaries now the `get_cost` returns a dict of the shape `{Items.Wood: 15}` so the fix would be `enough(c, costs[c] * n)`
